.github/workflows/test_frameworks.yml

```diff
@@ -157,24 +155,24 @@ jobs:
         run: pip3 install ../ddtrace
       - name: Run tests
         if: needs.needs-run.outputs.outcome == 'success'
-        run: ddtrace-run pytest -k "not test_no_exceptions_when_cancel_pending_request and not test_add_signal and not test_ode_removes and not test_skip_touchup and not test_dispatch_signal_triggers and not test_keep_alive_connection_context and not test_redirect_with_params and not test_keep_alive_client_timeout and not test_logger_vhosts and not test_ssl_in_multiprocess_mode"
+        run: ddtrace-run pytest -k "not test_reloader and not test_reload_listeners and not test_no_exceptions_when_cancel_pending_request and not test_add_signal and not test_ode_removes and not test_skip_touchup and not test_dispatch_signal_triggers and not test_keep_alive_connection_context and not test_redirect_with_params and not test_keep_alive_client_timeout and not test_logger_vhosts and not test_ssl_in_multiprocess_mode"
 
-  django-testsuite-3_1:
+  django-testsuite:
     strategy:
       matrix:
         include:
-          - suffix: DI profiler
-            expl_profiler: 1
-            expl_coverage: 0
-            profiling: 1
-            iast: 0
-            appsec: 0
-          - suffix: DI coverage
-            expl_profiler: 0
-            expl_coverage: 1
-            profiling: 1
-            iast: 0
-            appsec: 0
+          #- suffix: DI profiler
+          #  expl_profiler: 1
+          #  expl_coverage: 0
+          #  profiling: 1
+          #  iast: 0
+          #  appsec: 0
+          #- suffix: DI coverage
+          #  expl_profiler: 0
+          #  expl_coverage: 1
+          #  profiling: 1
+          #  iast: 0
+          #  appsec: 0
```

Comment:
What's the plan with these?

Reply:
I'll ask subject-matter experts to look into them